src/pages/persona/PersonaAnalysisResult.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import * as S from './styled';
+
+type PersonaAnalysisResultPageProps = {};
+
+enum TEXT {
+  BUTTON_DESCRIPTION = '체크리스트로 더 현명하게 방을 구해보자!',
+  GO_CHECKLIST = '나만의 체크리스트 보러가기',
+  SHARE = '체크해방 공유하기',
+}
+
+export default function PersonaAnalysisResultPage(_: PersonaAnalysisResultPageProps): ReactElement {
```

Comment:
P3;

아래처럼 바꿔주시면 좋을 것 같습니다.
저희 default param은 일단 없으면 안 넣기로 했습니다.
 더불어 type 대신 interface로 부탁드립니다

```suggestion
export default function PersonaAnalysisResultPage(): ReactElement {
```